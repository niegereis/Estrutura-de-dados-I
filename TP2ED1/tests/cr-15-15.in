15 15
r
***************
*   *   *     *
*** * * * *** *
*M* * *     * *
* * *** ***** *
* *   * *     *
* *** * * *****
*     * * *   *
* ***** * * * *
* *     *   * *
* *** ******* *
*   *     *   *
*** ******* * *
*           *  
***************