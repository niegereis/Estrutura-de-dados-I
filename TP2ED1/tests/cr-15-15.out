41
***************
*ooo*   *     *
***o* * * *** *
*M*o* *     * *
*o*o*** ***** *
*o*ooo* *     *
*o***o* * *****
*ooooo* * *   *
*o***** * * * *
*o*     *   * *
*o*** ******* *
*ooo*     *ooo*
***o*******o*o*
*  ooooooooo*oo
***************
